src/tsconfig.ce.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../out-tsc/app",
+    "baseUrl": "./",
+    "types": [
+      "node"
+    ]
+  },
+  "files": [
+    "main.ts",
+    "polyfills.ts"
+  ],
+  "exclude": [
+    "test*.ts",
+    "**/*.spec.ts",
+    "**/testing/**",
+    "app/dynamic/enterprise/**"
```

Comment:
In the community edition all of the enterprise edition sources will be excluded.